utils/s2n_init.c

```diff
@@ -77,10 +77,10 @@ static bool s2n_cleanup_atexit_impl(void)
 {
     /* all of these should run, regardless of result, but the
      * values to need to be consumed to prevent warnings */
+    s2n_wipe_static_configs();
```

Comment:
What does this change do?

Reply:
The `s2n_wipe_static_configs` function uses the memory callbacks to free the default configs. This needs to be done before calling `s2n_mem_cleanup` since [that resets memory callbacks](https://github.com/aws/s2n-tls/blob/5a3b1ad01d0ebde3e3c55d5d4a43242458424ca5/utils/s2n_mem.c#L69).

Reply:
This probably deserves a comment